robottelo/download_file.py

```diff
@@ -0,0 +1,32 @@
+import os
+import requests
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo import ssh
+
+
+def on_local(url, local_path=None, file_name=None):
+    """ Download file to local from url """
+    _, file_name = file_name or os.path.split(url)
+    if local_path is None:
+        local_path = '/tmp/'
+    r = requests.get(url)
+    with open('{}{}'.format(local_path, file_name), 'wb') as f:
+        for chunk in r.iter_content(chunk_size=1024):
+            if chunk:
```

Comment:
```suggestion
            if not chunk:
                     raise Exception
                     break
            f.write(chunk)
```

Reply:
Removed it from code. Not valid any more.